src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -237,10 +238,14 @@ export default {
 				: 'Support a featured borrower with a microloan.';
 		},
 		featuredLoanTitle() {
-			return 'A loan ending soon';
+			return this.isLoggedIn
+				? 'Recommended for you,'
+				: 'A loan ending soon';
 		},
 		featuredLoanSubtitle() {
-			return 'Make a difference for these borrowers who only have a short time remaining.'; // eslint-disable-line max-len
+			return this.isLoggedIn
+				? 'Make a difference for borrowers who only have a small amount remaining.'
```

Comment:
This message isn't correct, right? We are currently showing first recommended, not with small amount remaining.

Reply:
Agreed! But is the copy we have in the mocks, https://www.figma.com/file/6RV1BiJR8s8rqAVlv6dRwl/Lending-Home-V4---Featured-Loan?node-id=3145%3A263&mode=dev

Reply:
Okay I'll approve, but please include that question to Nathan as well in our slack channel.